src/SongManager.cpp

```diff
@@ -415,6 +439,59 @@ void SongManager::LoadSongDir( RString sDir, LoadingWindow *ld, bool onlyAdditio
 
 		SongPointerVector& index_entry = m_mapSongGroupIndex[sGroupDirName];
 		RString group_base_name= Basename(sGroupDirName);
+
+		RString sGroupIniPath = sDir + sGroupDirName + "/pack.ini";

```

Comment:
I think it should have a capital letter at the beginning. That's in line with `Portable.ini`.